.circleci/config.yml

```diff
@@ -5,32 +5,43 @@ orbs:
 jobs:
   update-helm-charts-index:
     docker:
-      - image: docker.mirror.hashicorp.services/circleci/golang:1.19.2
+      - image: docker.mirror.hashicorp.services/cimg/go:1.19.2
     steps:
       - checkout
       - run:
           name: verify Chart version matches tag version
+          environment:
+            RELEASE_TAG: << pipeline.parameters.release-tag >>
           command: |
-            GO111MODULE=on go get github.com/mikefarah/yq/v2
-            git_tag=$(echo "${CIRCLE_TAG#v}")
+            go install github.com/mikefarah/yq/v2@latest
+            export TAG=${RELEASE_TAG:-$CIRCLE_TAG}
+            git_tag=$(echo "${TAG#v}")
             chart_tag=$(yq r Chart.yaml version)
             if [ "${git_tag}" != "${chart_tag}" ]; then
               echo "chart version (${chart_tag}) did not match git version (${git_tag})"
               exit 1
             fi
       - run:
           name: update helm-charts index
+          environment:
+            RELEASE_TAG: << pipeline.parameters.release-tag >>
           command: |
             curl --show-error --silent --fail --user "${CIRCLE_TOKEN}:" \
                 -X POST \
                 -H 'Content-Type: application/json' \
                 -H 'Accept: application/json' \
-                -d "{\"branch\": \"main\",\"parameters\":{\"SOURCE_REPO\": \"${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}\",\"SOURCE_TAG\": \"${CIRCLE_TAG}\"}}" \
+                -d "{\"branch\": \"main\",\"parameters\":{\"SOURCE_REPO\": \"${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}\",\"SOURCE_TAG\": \"${RELEASE_TAG}\"}}" \
```

Comment:
Will `RELEASE_TAG` be set when this is not manually triggered? I wonder if we need to stash `TAG` in `$BASH_ENV` up above.

Reply:
Oops, good catch. I repeated the same `:-$CIRCLE_TAG` defaulting in this step instead, which should also work